recipes/Cbc/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "coin-or-cbc" %}
+{% set version = "2.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/coin-or/Cbc/archive/releases/{{ version }}.tar.gz
+    sha256: 601a7084e75955f98ccf1ef982d272cbe7f34558c527dadc661d49e6f0fc58a3
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make  # [unix]
```

Comment:
May also need some other utilities in some cases, but I guess they can be added as they come up. `patch` and dos2unix` come to mind.